app/controllers/items_controller.rb

```diff
@@ -16,6 +16,30 @@ def show
   def new
     @item = Item.new
     @user = current_user
+    # @item.item_category_id = params[:item_category_id]
+    
+    @measurement_options = []
+    Measurement.all.each do |measurement|
+      @measurement_options << measurement.unit
+    end
+
+    @category_options = []
+    ItemCategory.all.each do |category|
+      @category_options << category.name
+    end
+
+    # def return_item_id_from_item_name(item_category_name)
```

Comment:
remove this commented function, if it is no longer used.

Reply:
yep